### PR TITLE
43 free at shutdown

### PR DESCRIPTION
**Pull request by arpd**
These are all of the `sp_node_t*` lists that I could find.

### CODE REVIEW

**Comment by jvoisin** [comment]
Wow, strangers on the internet contributing to this little project of ours, amazing!

I'll review this as soon as I arrive at work tomorrow morning, thank you very much♥

**Comment by arpd** [comment]
@jvoisin Your issue[s] tagged "good first issue" pulled me in.

File: src/snuffleupagus.c [hunk 1]
```diff
@@ -118,10 +118,38 @@ PHP_MSHUTDOWN_FUNCTION(snuffleupagus) {
   pefree(SNUFFLEUPAGUS_G(config.config_upload_validation), 1);
   pefree(SNUFFLEUPAGUS_G(config.config_cookie_encryption), 1);
 
-  sp_list_free(SNUFFLEUPAGUS_G(config.config_disabled_functions->disabled_functions));
+  sp_node_t* disabled_functions = SNUFFLEUPAGUS_G(config.config_disabled_functions->disabled_functions);
+  sp_node_t* disabled_functions_ret = SNUFFLEUPAGUS_G(config.config_disabled_functions_ret->disabled_functions);
+  sp_node_t* disabled_constructs = SNUFFLEUPAGUS_G(config.config_disabled_constructs->construct_include);
+
+  sp_node_t* cursor = disabled_functions;
+  while(cursor) {
```
Comment (jvoisin) [inline src/snuffleupagus.c]: I think that those functions could be factorised into a single one :)

**Comment by jvoisin** [comment]
@blotus This looks good to me, if it's good for you too, please merge :)

@arpd Again, thank you very much ♥

**Pull request merged**